### PR TITLE
Chi 1316 childline zimbabwe staging

### PR DESCRIPTION
**Pull request by acedeywin**
Primary reviewer: @GPaoloni 

## Description

- New feature: Childline Zimbabwe Staging Setup

### Checklist
- [x] Corresponding issue has been opened
- [N/A] New tests added
- [N/A] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes CHI-1316

### CODE REVIEW

File: .github/workflows/zimbabwe_staging.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      # Call main-action to compile and deploy
+      - name: Executing main-action
+        uses: ./.github/actions/main-action
+        with:
+          account-sid: ${{ secrets.Zw-v1_STG_ACCOUNT_SID }}
+          auth-token: ${{ secrets.Zw-v1_STG_AUTH_TOKEN }}
```
Comment (GPaoloni) [inline .github/workflows/zimbabwe_staging.yml:33]: This two are not really following the naming we use for this, as we don't need "versions", just the country code. Could you change that?
Example
Screenshot from 2022-07-25 17-24-11

File: twilio-iac/terraform-poc-account/main.tf [hunk 2]
```diff
@@ -107,7 +107,6 @@ module aws_monitoring {
 }
 
 module github {
-
```
Comment (GPaoloni) [inline twilio-iac/terraform-poc-account/main.tf:110]: Thanks, this one bugged me and forgot to change haha

File: .github/workflows/zimbabwe_staging.yml (hunk 1 above)
Comment (GPaoloni) [reply .github/workflows/zimbabwe_staging.yml:33]: This secrets are created by the `github` terraform module, and after checking they are there, with the proper name, so you'll only need to change the name here in this workflow and reference the right one.
Screenshot from 2022-07-25 17-32-21

**Review by GPaoloni — APPROVED** [review]
Just one small comment, but other than that looks good to me! Approving as I don't think this needs a re-review after you do the change :)

File: twilio-iac/terraform-poc-account/main.tf (hunk 2 above)
Comment (acedeywin) [reply twilio-iac/terraform-poc-account/main.tf:110]: @GPaoloni That was a clumsy mistake. Fixed.

**Pull request merged**